docs/articles/nunit-engine/extensions/Installing-Extensions.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: installingextensions
+---
+
+# Installing Engine Extensions
+
+Extensions are located by the engine by use of an `.addins` file.
+
+For certain package managers, default `.addins` have been created such that when both engine and extensions are installed from the same package manager, all extensions are installed automatically. Currently, this includes NuGet and Chocolatey.
```

Comment:
The wording "all extensions are installed automatically" seems misleading. No extensions are installed automatically, i.e. the user has to install them using the same package manager.

Suggest "... default `.addins` files are included, so that any extensions installed using the same package manager are are automatically used by the engine."

Reply:
Good clarification, I like how you've worded it. Thanks!